proxy/proxy.go

```diff
@@ -350,11 +352,26 @@ func initLogging() {
 	}
 }
 
+var enablePprof bool
+
+func setupPprof() {
+	if !enablePprof {
+		return
+	}
+
+	go func() {
+		http.ListenAndServe("localhost:6060", nil)
```

Comment:
I'm saying `6060` isn't in `/etc/services`, so this number seems to have been randomly chosen. So, what if another application running on the same machine also tries to use the same port number? If it's not a well-known port number (or even if it is infact), there should be a way to change it.